app/views/support/ContactCentreOps.scala

```diff
@@ -12,7 +12,7 @@ object ContactCentreOps {
 
   def phone(contactUsCountry: Option[Country] = None): String = {
     if (contactUsCountry exists countriesHandledByUSCallCentre)
-      s"1-844-632-2010 (toll free); ${directLine(contactUsCountry)} (direct line). Lines are open 9:15am-6pm, Monday to Friday (EST/EDT)"
```

Comment:
is `EDT` no longer a thing?

Reply:
Although it is missing from the copy I think its pragmatic to add it - we have BST for the UK number